src/db/tables/subscribers.js

```diff
@@ -8,6 +8,7 @@ import AppConstants from '../../appConstants.js'
 
 const knex = createDbConnection();
 const { DELETE_UNVERIFIED_SUBSCRIBERS_TIMER } = AppConstants
+const MONITOR_PREMIUM_CAPABILITY = "monitor";
```

Comment:
Was going to ask if this should be `MONITOR_PLUS_CAPABILITY` but looks like we use this in 3&times; places.
Although not sure if it is worth extracting into somewhere central and exporting:
```sh
git grep -Pn "\bMONITOR_[A-Z_]+\b =" | cat

src/app/api/v1/fxa-rp-events/route.ts:36:const MONITOR_PREMIUM_CAPABILITY = "monitor";
src/app/functions/server/changeSubscription.ts:8:const MONITOR_PREMIUM_CAPABILITY = "monitor";
src/db/tables/subscribers.js:11:const MONITOR_PREMIUM_CAPABILITY = "monitor";
```